### PR TITLE
Added testcase C5 to the file readingsCompareMeterQuantity

### PR DESCRIPTION
**Pull request by LauraV-702**
# Description
Completed and added testcase C5 into the readingsCompareMeterQuantity.js file

Team:
@LauraV-702
@alstondsouza1
@Lizock
@Z1springer

Fixes testcase c5 from issue #962

## Type of change
- [ ] Note merging this changes the database configuration.
- [x] This change requires a documentation update

## Checklist
- [x] I have followed the [OED pull request](https://openenergydashboard.org/developer/pr/) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.org/developer/cla/) and each author is listed in the Description section.

## Limitations

### CODE REVIEW

File: src/server/test/web/readingsCompareMeterQuantity.js [hunk 1]
```diff
@@ -68,9 +68,22 @@ mocha.describe('readings API', () => {
...
 
-				// Add C5 here
+				mocha.it('C5: 7 day shift end 2022-11-01 15:00:00 (beyond data) for 15 minute reading intervals and quantity units & kWh as kWh', async () => {
+					await prepareTest(unitDatakWh, conversionDatakWh, meterDatakWh);
+					const unitId = await getUnitId('kWh');
+					const expected = [9132.81261972035, 13147.7382388332];
+					const res = await chai.request(app).get(`/api/compareReadings/meters/${METER_ID}`)
+					  .query({
+						curr_start: '2022-10-30 00:00:00',
+						curr_end: '2022-11-01 15:00:00',
```
Comment (huss) [inline src/server/test/web/readingsCompareMeterQuantity.js]: This line has extra spaces at the end that need to be removed.

File: src/server/test/web/readingsCompareMeterQuantity.js [hunk 2]
```diff
@@ -68,9 +68,22 @@ mocha.describe('readings API', () => {
...
+					const expected = [9132.81261972035, 13147.7382388332];
+					const res = await chai.request(app).get(`/api/compareReadings/meters/${METER_ID}`)
+					  .query({
+						curr_start: '2022-10-30 00:00:00',
+						curr_end: '2022-11-01 15:00:00',  
+						shift: 'P7D',  
+						graphicUnitId: unitId
+					  });
+					expectCompareToEqualExpected(res, expected);
+				  });
```
Comment (huss) [inline src/server/test/web/readingsCompareMeterQuantity.js]: This line has extra tabs and spaces at the end that need to be removed.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @LauraV-702, @alstondsouza1, @Lizock & @Z1springer for their first contribution to OED. Review and testing found it works as desired. I made two small comments about extra white space at the end of lines. I'm unsure why the OED checks did not catch this. Also, my records do not show the CLA being signed by @alstondsouza1, @Lizock & @Z1springer. Could you please do this via the link in the description or let me know that my records are off.

**Comment by LauraV-702** [comment]
Hello, I've made changes as you directed, thank you! 
Hopefully these work this time. I've also notified my teammates to sign the CLA, I'll be sure to double check with them once more.

**Comment by huss** [comment]
> Hello, I've made changes as you directed, thank you! Hopefully these work this time. I've also notified my teammates to sign the CLA, I'll be sure to double check with them once more.

@LauraV-702 It all looks good. One more CLA and this can be accepted. Thanks for addressing the comments.

**Comment by LauraV-702** [comment]
@huss Thanks for the confirmation! I've double checked with my team mates and now they've all signed the CLA!

**Review by huss — APPROVED** [review]
Everything looks good. Congratulations to @LauraV-702, @alstondsouza1, @Lizock & @Z1springer on your first accepted contribution to OED.

**Comment by LauraV-702** [comment]
Awesome, thank you!

**Pull request merged**